R/table.R

```diff
@@ -132,7 +132,17 @@ armadillo.copy_table <- # nolint
 #'
 #' @export
 armadillo.load_table <- function(project, folder, name) { # nolint
```

Comment:
I've tested this and it works. However it's not very generic - e.g. if we want to expand this to loading other types of object. I'm happy to approve for now, but maybe next time we want to extend this to other object types we should think about refactoring so we can more easily add support for other types?

Reply:
That's valid. We do however really have to think about how to make it more generic. Possibly pass a loadfunction to it then, don't know. Maybe we can create a story for it? I think for now this is good enough, but I do agree that it's a bit limited.